src/transformers/models/llava/modeling_llava.py

```diff
@@ -360,16 +360,17 @@ def forward(
         ```python
         >>> from PIL import Image
         >>> import requests
-        >>> from transformers import AutoProcessor, LlavaForConditionalGeneration
+        >>> from transformers import AutoProcessor, AutoTokenizer, LlavaForConditionalGeneration
```

Comment:
Rather than add the tokenizer here, instead we should update L377 to use the processor

Reply:
Question, if without tokenier, how can we decode the outputs?

Reply:
You can call batch_decode on the processor (it just calls the tokenizer under the hood): `processor.batch_decode(token_ids)`